libs/sdk-core/src/lib/db/kb/kb.models.ts

```diff
@@ -233,6 +233,7 @@ export interface WidgetFeatures {
   knowledgeGraph?: boolean;
   useSynonyms?: boolean;
   autofilter?: boolean;
+  noBM25forChat?: boolean;
```

Comment:
I think the `chat` endpoint is used to show the first page of results.
Will this option remove bm25 from the search results ?

Reply:
Yes it will.

We are aware this option might be not entirely consistent (and by default it will be false), but Eudald needs it for demo purpose.